pkg/networkmanager/interfaces.js

```diff
@@ -871,7 +871,7 @@ export function NetworkManagerModel() {
                     return call_object_method(self,
                                               "org.freedesktop.NetworkManager.Settings.Connection", "Update",
                                               settings_to_nm(settings, priv(self).orig))
-                            .then(function () {
+                            .then(results => {
                                 set_settings(self, settings);
                             });
```

Comment:
Does this actually do anything? `results` looks unused here. Should just be `.then(() => set_settings(self, settings))`?

Reply:
It doesn't change anything, but it follows the (new) convention that call_object_method is always used like `call_object_method(...).then(results => ...)`. That is supposed to make it clear when the "then" function gets.